From 170e72ed3a5b413783b5cb9dc135d8b6a408de93 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sat, 25 Apr 2020 14:13:43 +0200
Subject: [PATCH 5/5] Declare libgsettings-qt dependency for cpptest &
 gsettings-qt

Resolves cpptest & gsettings-qt sometimes being linked before
libgsettings-qt was built which resulted in an error.

Fixes #2
---
 gsettings-qt.pro | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/gsettings-qt.pro b/gsettings-qt.pro
index 15e3684..b7dbbbf 100644
--- a/gsettings-qt.pro
+++ b/gsettings-qt.pro
@@ -1,2 +1,14 @@
 TEMPLATE = subdirs
-SUBDIRS += src/gsettings-qt.pro GSettings/gsettings-qt.pro tests/tests.pro tests/cpptest.pro
+SUBDIRS += \
+    libgsettings-qt \
+    gsettings-qt \
+    tests \
+    cpptest
+
+libgsettings-qt.file = src/gsettings-qt.pro
+gsettings-qt.file = GSettings/gsettings-qt.pro
+gsettings-qt.depends = libgsettings-qt
+
+tests.file = tests/tests.pro
+cpptest.file = tests/cpptest.pro
+cpptest.depends = libgsettings-qt
-- 
2.26.2

